sparse/var/lib/flash-partition/device-info

```diff
@@ -0,0 +1,7 @@
+CPUCHECK_STRING="Qualcomm MSM 8274AB"
```

Comment:
cat /proc/cpuinfo Reveals:

Mi3: Hardware        : Qualcomm MSM8974PRO-AB
Mi4: Hardware        : Qualcomm MSM8974PRO-AC

So....

<mal> Nokius: there are two revisions of fp2, one with AB and other was maybe AA or AC
<Nokius> :D
<mal> so I just ignored the last letter
<mal> :D